python/openmldb/dbapi/dbapi.py

```diff
@@ -505,8 +506,8 @@ def close(self):
         raise NotSupportedError("Unsupported in OpenMLDB")
 
     def cursor(self):
-        return Cursor(self._db, self._zk, self._zkPath, self)
+        return Cursor(self._db, self)
 
 # Constructor for creating connection to db
-def connect(db, zk, zkPath):
-    return Connection(db, zk, zkPath)
+def connect(db,*args,is_cluster_mode=True):
```

Comment:
Format the code and add spaces between the parameters.

Reply:
updated.